content/blog/2022/06/Optimizing-Camel-K-Integration-Build-Time/index.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "Optimizing Camel-K Integration Build Time"
+date: 2022-06-24
+draft: true
```

Comment:
This has to be removed for the post to be published :)

Reply:
sorry, slipped out.